### ISSUE TITLE
Accessibility Audit: RST website review

### ISSUE THREAD

**Issue by JLWade** [bcgov/nr-rec-resources#114]
**Describe the task**
Accessibility audit of sitesandtrailsbc.ca. This will help us build some baseline metrics of accessibility now versus once we have a new website. Our goal is to reach WCAG Level AA and readability level AAA. 
https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-standards 

**Acceptance Criteria**
- [ ] Accessibility report is added to confluence: https://apps.nrs.gov.bc.ca/int/confluence/display/BCPRS/Accessibility+Audits
- [ ] List of bugs/fixes is attached to report and also saved in the same location

**Additional context**

**Comment by JLWade** [comment]
@manuji also checking to see the status of this ticket?
We can carry both over to the next sprint (starting on thursday) if necessary but wanted to check.

**Comment by JLWade** [comment]
will do! thanks :)

**Comment by manuji** [comment]
The accessibility review has been completed on https://www.sitesandtrailsbc.ca/ against the WCAG 2.2 AA and for Reading level AAA.

Please not that the other Level AAA criteria except Reading level AAA has not been checked. 

**Summary**

Passed: 25
Failed: 20
Cannot tell: 0
Not present: 11
Not checked: 30

Refer to the Evaluation report here: 

[Accessibility evaluation report for Recreation Sites and Trails BC - Legacy site.pdf](https://github.com/user-attachments/files/18517592/Accessibility.evaluation.report.for.Recreation.Sites.and.Trails.BC.-.Legacy.site.pdf)

Refer to the bug report here: 

[Accessibility bug report for Recreations Sites and Trails BC - Legacy site..pdf](https://github.com/user-attachments/files/18517595/Accessibility.bug.report.for.Recreations.Sites.and.Trails.BC.-.Legacy.site.pdf)

**Comment by JLWade** [comment]
thanks @manuji moving this one to done!
